index.html

```diff
@@ -0,0 +1,297 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.css">
+    </link>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Serif:opsz@8..144&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <nav class="navbar">
+        <header><a class="header">John Doe </a></header>
+        <img src="./images/menu.svg" alt="menusvg" class="navbarmenu" />
+        <ul class="itemslist">
+            <li class="listitem">Portfolio</li>
+            <li class="listitem"><a>About</a></li>
+            <li class="listitem"><a>Contact</a></li>
+            <li class="listitem"><a><img src="./images/contact.svg" alt="" /></a></li>
+        </ul>
+    </nav>
+    <main>
+        <section class="textsection">
+            <div class="headercontent">
+                <h2 class="headtext"> Hey There. I’m Alex</h2>
+                <p class="developer">I’am a Software Developer</p>
+                <p class="build">I can help you build a product , feature or website Look through some of my work and
+                    experience! If you
+                    like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+            </div>
+            <div class="icons">
+                <ul class="iconscontainer1">
+                    <li><img src="./images/Icon - Linkedin.svg" alt="linkedin" class="linkedin" /></li>
+                    <li><img src="./images/Icon - Angelist.svg" alt="angelist" class="angelist" /></li>
+                    <li><img src="./images/Icon - Medium.svg" alt="medium" class="medium" /></li>
+                    <li><img src="./images/Icon - Twitter.svg" alt="twitter" class="twitter" /></li>
+                    <li><img src="./images/Github.svg" alt="github" class="github" /></li>
+                </ul>
+            </div>
+            <div class="icons2">
+                <ul class="iconscontainer2">
+                    <li><img src="./images/Icon - Linkedin.svg" alt="linkedin" class="linkedin" /></li>
+                    <li><img src="./images/Icon - Angelist.svg" alt="angelist" class="angelist" /></li>
+                    <li><img src="./images/Icon - Medium.svg" alt="medium" class="medium" /></li>
+                    <li><img src="./images/Icon - Twitter.svg" alt="twitter" class="twitter" /></li>
+                    <li><img src="./images/Github.svg" alt="github" class="github" /></li>
+                </ul>
```

Comment:
- [ ] Please remove this part because is repeated code which is a not good practice. I think you can use CSS and media query to make your icon fit both mobile and desktop versions. :+1: 

- [ ] Please make sure your design fit the desktop version you can see a black space when we scrolled to the right which affected the design. Please fix and make your design similar to the Figma template :smile: 
#### Please check the recorded video below
https://www.loom.com/share/7667f0232dbd4d07a8f4a200aaef34b5